### PR TITLE
EFA-656 Do not append messages that are already in remote folder

### PR DESCRIPTION
**Pull request by ig-garcia**

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java [hunk 1]
```diff
@@ -153,6 +151,8 @@ public class MessagingController implements Sync.MessageToSendCallback {
     public static final long INVALID_MESSAGE_ID = -1;
     public static final long SHARE_SIZE_THRESHOLD = 64000;
     public static final int SHARE_MAX_FILENAME_SIZE = 20;
+    public static final int CHECK_MESSAGE_ON_SERVER_INTERVAL = 5000;
```
Comment (thgoebel-planck) [inline k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java:154]:
```suggestion
    public static final int CHECK_MESSAGE_ON_SERVER_INTERVAL_MILLIS = 5000;
```

I like having the unit in constant names. Or comment the unit somewhere.

File: k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java [hunk 2]
```diff
@@ -3397,14 +3397,17 @@ private void moveOrDeleteSentMessage(Account account, LocalStore localStore,
...
-            processPendingCommands(account);
-
-            Timber.i("Moved sent message to folder '%s' (%d)",
-                    account.getSentFolderName(), localSentFolder.getId());
-
+            if (isLocalMessageAlreadyOnServer(account, encryptedMessage.getMessageId(), account.getSentFolderName())) {
+                Timber.d("EFA-656 Sent message is already in remote folder, nothing to do");
+            } else {
+                Timber.d("EFA-656 Sent message not found on remote folder, appending it");
+                //localFolder.moveMessages(Collections.singletonList(message), localSentFolder);
```
Comment (thgoebel-planck) [inline k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java:3404]:
```suggestion
```

You removed the function parameter, so no real need to keep this line around imho. Also, it is already appended to the localSentFolder above.

**Review by thgoebel-planck — APPROVED** [review]
To small things that would be nice to have. Otherwise good.

File: k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java (hunk 2 above)
Comment (ig-garcia) [reply k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java:3402]: Removed, thanks.

File: k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java (hunk 1 above)
Comment (ig-garcia) [reply k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java]: Constants were removed.

**Pull request merged**